StaticAnalyzer/views/ios/plist_analysis.py

```diff
@@ -17,11 +18,14 @@
 
 def convert_bin_xml(bin_xml_file):
     """Convert Binary XML to Readable XML"""
-    plist_obj = readPlist(bin_xml_file)
-    data = writePlistToString(plist_obj)
-    return data
-
-
+    try:
+      plist_obj = readPlist(bin_xml_file)
+      data = writePlistToString(plist_obj)
+      return data
+    except biplist.InvalidPlistException:
+      logger.warning("Failed to convert plist")
+    
```

Comment:
Will it be fixed?
Do you want to return an empty string?

Reply:
for the moment it only prevent to stop analysis